docs/articles/professional-services.md

```diff
@@ -0,0 +1,40 @@
+# Professional Services
+Evolve your products and apps to expand your markets. Nebular Services is ready to help speed up the evolution of your business. 
+The Nebular team is ready to bring you their expertise with following services.
+
+### Components customization
+Let Akveo’s experts help you get more out of your Nebular experience. Modify, extend, and tailor Nebular components and controls to fit your needs. Our developers know the products inside and out, so there's no need to spend hours searching for a solution. We create themes, modules, and components to match your company branding.
+
+### Custom development 
+Do you want to make a web or mobile app? Or even extend an existing one with new features?
+Our performance specialists will analyze your product and offer an optimized solution based on your actual needs. We will work with you to create an amazing product that delivers.
+
+### Consulting
+Sometimes effective troubleshooters are hard to come by. And sometimes, a little boost at just the right time is all it takes to bring your project to the next level. Get our comprehensive support and level up your project. Here's how we can help:
+- Code Review/Analysis 
+- Debugging and Programming Help
+- Deployment 
+- Workshop Mentoring
+ 
+<a href="mailto:contact@akveo.com?subject=Nebular services consulting request">Want to know more?</a>
+
+<hr>
+## Support 
+The mission of the Akveo support team is to help you to solve technical challenges,use best practices with Nebular, and employ the most effective programming techniques for your project. If you need an extra boost, we can quickly become an extension of your team, ensuring your business has all the strategic support it needs. Our Nebular Support Service offers a number of solutions to meet your support level and needs:
+- Assured response to every client request
+- Debugging
+- Deployment consulting
+- Remote assistance from an engineer 
+- Full application code analysis 
+
+<a href="mailto:contact@akveo.com?subject=Nebular services support request">Need help? Contact us</a>
+
+
+<hr>
+## Trainings 
+Get the practical knowledge your team needs to effectively use Nebular. 
+ 
+Online study. The convenience of desktop learning from anywhere. Our Nebular tutors offer a comprehensive online training course for one very affordable price. The course lets you gain enough experience to create an amazing app based on a Nebular framework, developed on Angular technology.
+Offline study.  Onsite training. Wherever you are, ur Nebular expert will come to you and boost the skills of your developers, quickly and effectively. . We will tailor a high-impact training course that addresses your specific goals and gets your team’s skills where they need to be.
```

Comment:
Two dots here:
`... quickly and effectively. . We will tailor ...`

Reply:
good catch